deluge/ui/web/js/deluge-all/TorrentGrid.js

```diff
@@ -67,15 +67,27 @@
     }
 
     function etaSorter(eta) {
-        return eta * -1;
+        if (eta < 0) {
```

Comment:
See using guard clauses below, perhaps putting the 0 assertion first, since perhaps more likely outcome